src/Exception/RequestException.php

```diff
@@ -91,7 +91,7 @@ public static function create(
         }
 
         $uri = $request->getUri();
-        $uri = static::obfuscateUri($uri);
+        $uri = self::obfuscateUri($uri);
```

Comment:
This could potentially be wrong if it is intended to allow child classes to override `obfuscateUri()` and allow that to be applied with this factory method, if that is the case, then the visibility should be loosened to either `protected` or `public`

Reply:
this is a breaking change. please leave this alone. we can change on guzzle 9